cmd/common/progress/progress.go

```diff
@@ -1,12 +1,52 @@
-package common
+//nolint:revive
+package progress
 
 import (
 	"fmt"
 	"io"
+	"math"
 	"os"
+	"strings"
 	"time"
 )
 
+// PrintProgressBar prints a nice progress bar.
+func PrintProgressBar(f io.Writer, msg string, done uint64, total uint64, final bool) {
```

Comment:
Could you use the percentage on the left, then the barline and on the right amount of data in MiB?

Something like wget:

<img width="925" height="40" alt="image" src="https://github.com/user-attachments/assets/c11e88ce-ba84-4dbc-9d40-27db614af1a5" />

Later we can also add the current speed and an eta.